src/Core/Managed/Shared/Channel/Transmission.cs

```diff
@@ -221,6 +238,38 @@ protected virtual WebRequest CreateRequest(Uri address)
             return request;
         }
 
+#if NET40
+        private Task<HttpWebResponseWrapper> SendRequestAsync(WebRequest request)
+        {
+            return Task.Factory.FromAsync(request.BeginGetRequestStream, request.EndGetRequestStream, null)
+                .ContinueWith(
+                    task =>
```

Comment:
Renamed to GetResponseAsync. This method is implemented as async method for NET45 so I didn't remove Async suffix.